webapp/src/ts/services/target-aggregates.service.ts

```diff
@@ -1,6 +1,7 @@
 import { Injectable, NgZone } from '@angular/core';
 import * as moment from 'moment';
 import { isString as _isString } from 'lodash-es';
+import { Contact } from '@medic/cht-datasource/dist/libs/contact';
```

Comment:
@jkuester is it okay to import the type directly from here?

Reply:
Probably not a bid deal either way, but for the sake of consistency and completeness it would be better to export the interface from the cht-datasource index.ts (similar to what we did [here](https://github.com/medic/cht-core/blob/282faee191a448bbcff8f67baa8b3fc844ef14dc/shared-libs/cht-datasource/src/index.ts#L35) and then remove `@internal` from the JSDoc for `Contact` (can add a bit of real docs that indicate a contact could either be a place or a person...)

I think in the long-term types will be a bit more manageable if we can see which ones are explicitly exported.

Reply:
Got it! Fixed